sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 1.0.0b3 (Unreleased)
```

Comment:
Can you change Unreleased to the date of release?

Reply:
oh yeah, i'm sorry that totally slipped my mind

Reply:
I'm going with 03/10 tuesday bc this is what we discussed before

Reply:
Checked the spreadsheet, 3/10 looks correct